include/event2/util.h

```diff
@@ -76,6 +76,18 @@ extern "C" {
 
 #include <time.h>
 
+#ifdef __sun
+#ifdef SO_FLOW_NAME
+/* Since it's impossible to detect the Solaris 11.4 version via OS macros,
+ * so we check the presence of the socket option SO_FLOW_NAME that was first
+ * introduced to Solaris 11.4 and define a custom macro for determining 11.4.
+ */
+#define EVENT__SOLARIS_11_4 (1)
```

Comment:
Hm, but this check should be in realtime, since libevent may be compilied with 11.3, but used on 11.4 and vise-versa

It is OK for now to keep it as-is I guess, but worth a comment.

Reply:
Done.